artiq/master/scheduler.py

```diff
@@ -312,11 +311,6 @@ async def _do(self):
                 logger.error("got worker exception in analyze stage of RID %d."
                              " Results will still be saved.", run.rid)
```

Comment:
This "Results will still be saved" message should be removed, or there should be a similar message if the ``run`` stage terminates with an exception.